### PR TITLE
Fundraising UI tweaks

### PR DESCRIPTION
**Pull request by bpierre**
A WIP PR to integrate the new aragonUI components into Fundraising.

- [x] Fix aragonUI assets loading.
- [x] Move to use `Header`.
- [x] Fix the side panel transition on Firefox.
- [x] Reserve settings: use more components from aragonUI.
- [ ] Make the timeline its own component.

### CODE REVIEW

File: apps/aragon-fundraising/app/src/screens/Reserves.js [hunk 1]
```diff
@@ -221,123 +175,114 @@ export default () => {
...
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayRate} DAI / month
-              </Text>
-            </div>
-            <div css="display: flex; flex-direction: column; margin-bottom: 1.5rem;">
-              {NotificationLabel('Floor', hoverTextNotifications[2])}
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayFloor} DAI
-              </Text>
+    <React.Fragment>
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/screens/Reserves.js]: The linter prefers the shorthand version of Fragments `<></>`

File: apps/aragon-fundraising/app/src/screens/Reserves.js [hunk 2]
```diff
@@ -221,123 +175,114 @@ export default () => {
     }
   }
 
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const editMonthlyAllocationButton = <Button icon={<img src={EditIcon} />} label="Edit monthly allocation" onClick={() => setOpened(true)}></Button>
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/screens/Reserves.js]: The linter prefers self-closing when the component doesn't have children `<Button />`

File: apps/aragon-fundraising/app/src/screens/Reserves.js [hunk 3]
```diff
@@ -221,123 +175,114 @@ export default () => {
...
+                >
+                  Edit reserve settings
+                </h1>
+                <ReserveSetting label="Monthly allocation" helpContent={helpContent[0]} value={`${displayRate} DAI / month`} />
+                <ReserveSetting label="Floor" helpContent={helpContent[2]} value={`${displayFloor} DAI`} />
+                <div>{layoutName !== 'small' && editMonthlyAllocationButton}</div>
+              </div>
+              <div>
+                {[[daiSymbol, daiRatio], [antSymbol, antRatio]].map(([symbol, ratio], i) => (
+                  <ReserveSetting
```
Comment (deamme) [inline apps/aragon-fundraising/app/src/screens/Reserves.js]: Can we get these labels to align with the `Monthly allocation` and `Floor` labels?

Comment (bpierre) [reply apps/aragon-fundraising/app/src/screens/Reserves.js]: Oops, done! https://github.com/AragonBlack/fundraising/pull/114/commits/65615f5ca66f5e03d872a0b05619d42f7a0f909c

File: apps/aragon-fundraising/app/src/screens/Reserves.js (hunk 2 above)
Comment (bpierre) [reply apps/aragon-fundraising/app/src/screens/Reserves.js]: Done in https://github.com/AragonBlack/fundraising/pull/114/commits/7fefdbb7c7f70f62b47cc6d0087481f72cd61c2d

File: apps/aragon-fundraising/app/src/screens/Reserves.js (hunk 1 above)
Comment (bpierre) [reply apps/aragon-fundraising/app/src/screens/Reserves.js]: Done in https://github.com/AragonBlack/fundraising/pull/114/commits/7fefdbb7c7f70f62b47cc6d0087481f72cd61c2d

**Pull request merged**